dist/README.md

```diff
@@ -5,7 +5,7 @@ Weld
 What is it?
 ------------
 
-Weld is the reference implementation of CDI: Contexts and Dependency Injection 
+{CUSTOM_TEXT}: Contexts and Dependency Injection
```

Comment:
This is little hack. You can specify your custom text here during script execution (it is replaced with 'sed' command). If you leave the value empty, then the default text ("Weld is the reference implementation of CDI") with cdi version would be included.